datafusion/core/src/datasource/physical_plan/parquet/page_filter.rs

```diff
@@ -249,10 +260,7 @@ impl PagePruningPredicate {
 /// that `extract_page_index_push_down_predicates` only return
 /// predicate with one col)
 ///
-fn find_column_index(
-    predicate: &PruningPredicate,
-    row_group_metadata: &RowGroupMetaData,
-) -> Option<usize> {
+fn find_column_name(predicate: &PruningPredicate) -> Option<String> {
```

Comment:
This function docstring needs to be updated if we make this change. However I have a slightly different proposal here: https://github.com/manoj-inukolunu/arrow-datafusion/pull/1

Reply:
Thank you @alamb  I like the proposal its much cleaner. I have merged your pr.